src/herder/PendingEnvelopes.cpp

```diff
@@ -333,6 +338,9 @@ PendingEnvelopes::slotClosed(uint64 slotIndex)
 
         mTxSetFetcher.stopFetchingBelow(slotIndex + 1);
         mQuorumSetFetcher.stopFetchingBelow(slotIndex + 1);
+
```

Comment:
slotClosed in called on ledgerClosed
eraseBelow is called on processSCPQueue

I think that we could merge these two, but when doing this PR as didn't want to go too deep in the code to trace the paths in which these two methods work. I'll be happy to do this later.